src/client.js

```diff
@@ -421,43 +508,49 @@ class RPCClient extends EventEmitter {
   /**
    * Set current voice settings, overriding the current settings until this session disconnects.
    * This also locks the settings for any other rpc sessions which may be connected.
-   * @param {Object} args Settings
-   * @returns {Promise}
+   * @param {Partialize<VoiceSettings>} settings The new settings to use
```

Comment:
"Partialize" is not a thing in jsdoc.

Reply:
teating with a type - forgot to change back